build.sbt

```diff
@@ -88,10 +88,6 @@ lazy val docs = (projectMatrix in file("generated-docs")) // important: it must
     mdocExtraArguments := Seq("--clean-target"),
     publishArtifact := false,
     name := "docs",
-    libraryDependencies ++= Seq(
-      "com.softwaremill.sttp.client4" %% "cats" % "4.0.0-M6",
-      "org.typelevel" %% "cats-effect" % "3.5.2"
-    ),
```

Comment:
We keep all the dependencies in `Dependencies.scala` and I'd rather move it there too

Reply:
Currently, all dependencies are in `Dependencies.scala`. I've removed `libraryDependencies` from the docs module as they are now derived from other modules, in this case, from `fs2` module.

Reply:
Was it code removed, when I thought it's code added?... Ignore me 🤦